pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -1277,6 +1419,45 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, AumProtocolFeeCache,
         _require(tokenData != 0, Errors.INVALID_TOKEN);
     }
 
+    function _setCircuitBreakerRatio(
+        IERC20 token,
+        uint256 initialPrice,
+        uint256 minRatio,
+        uint256 maxRatio
+    ) internal {
+        // Has to be > minRatio (if equal, encoded value would be 0, indistinguishable from no circuit breaker)
+        _require(minRatio == 0 || minRatio > _MIN_CIRCUIT_BREAKER_RATIO, Errors.MIN_CIRCUIT_BREAKER_RATIO);
+        _require(maxRatio == 0 || maxRatio <= _MAX_CIRCUIT_BREAKER_RATIO, Errors.MAX_CIRCUIT_BREAKER_RATIO);
+        _require(maxRatio >= minRatio, Errors.INVALID_CIRCUIT_BREAKER_RATIOS);
```

Comment:
This line needs to be:
`_require(maxRatio >= minRatio || maxRatio == 0, Errors.INVALID_CIRCUIT_BREAKER_RATIOS);`
otherwise you cannot have a minRatio without a maxRatio